docs/devel.md

```diff
@@ -29,6 +29,17 @@ Unit tests can be run using `make test`
 
 Integration tests can be run using `make integration`
 
+
+### Code Coverage:
+
+Run first the tests.
+
+Then you can visualize the profile in html format:
+
+`go tool cover -html=cover.out`
```

Comment:
What about having a Makefile target `coverage` that depends on `test` and then does this? Or include this step directly when you run `make test`?

Reply:
we could include directly to make test, the thing is that it will open a page on browser. I dunno it can be maybe noise to have it each time and if circleci would work with it. we can try.

Or we can have a target `coverage`

Reply:
I would add a `coverage` target, with some`echo "You can see the coverage report in blah-blah-blah..."`